docs: Update docs on testing Electron apps with WebdriverIO (#40083)
electron/electron: docs/tutorial/automated-testing.md

> First you need to run the WebdriverIO starter toolkit in your project root directory:

Reviewer: I think it would be nice to note in the first paragraph at what WDIO version Electron support was added, in case a project is already using WDIO and needs to upgrade to get the latest capabilities.

Author: addressed in https://github.com/electron/electron/pull/40083/commits/a9849acd2604fcefaf2820e5ecee405f37c6bc48

```sh npm2yarn
npm init wdio@latest ./
```

This starts a configuration wizard that helps you put together the right setup, installs all necessary packages, and generates a `wdio.conf.js` configuration file. Make sure to select _"Desktop Testing - of Electron Applications"_ on one of the first questions asking _"What type of testing would you like to do?"_.

#### Connect WDIO to your Electron app

> After the configuration wizard finished running your `wdio.conf.js` should include roughly the following content:

Reviewer: ```suggestion
After running the configuration wizard, your `wdio.conf.js` should include roughly the following content:
```

```js title='wdio.conf.js' @ts-nocheck
export const config = {
  // ...
  services: ['electron'],
  capabilities: [{
    browserName: 'electron',
    'wdio:electronServiceOptions': {
      // WebdriverIO can automatically find your bundled application
      // if you use Electron Forge or electron-builder, otherwise you
      // can define it here, e.g.:
      // appBinaryPath: './path/to/bundled/application.exe',
      appArgs: ['foo', 'bar=baz']
    }
  }]
  // ...
}
```

#### Write your tests

> To interacting with elements on the screen you can use the [WebdriverIO API](https://webdriver.io/docs/api) that helps you to write concise and expressive tests. The framework provides custom "matchers" that make asserting the state of your application easy, e.g.:

Reviewer: ```suggestion
Use the [WebdriverIO API](https://webdriver.io/docs/api) to interact with elements on the screen. The framework provides custom "matchers" that make asserting the state of your application easy, e.g.:
```

```js @ts-nocheck
import { browser, $, expect } from '@wdio/globals'

describe('keyboard input', () => {
  it('should detect keyboard input', async () => {
    await browser.keys(['y', 'o'])
    await expect($('keypress-count')).toHaveText('YO')
  })
})
```

> Furthermore WebdriverIO allows you to access Electron APIs to get static information about your application:

Reviewer: ```suggestion
Furthermore, WebdriverIO allows you to access Electron APIs to get static information about your application:
```

$ npx wdio run wdio.conf.js
```

> WebdriverIO helps managing to launch and shut down the application for you.

Reviewer: ```suggestion
WebdriverIO helps launch and shut down the application for you.
```

> #### More Documentation

Reviewer: ```suggestion
#### More documentation
```
